### ISSUE TITLE
Specifying -d is with respect to the location of phpdoc.xml not the current working directory.

### ISSUE THREAD

**Issue by SJrX** [phpDocumentor/phpDocumentor2#1722]
```
$../phpDocumentor-2.8.5/bin/phpdoc parse -d ./application/ -c ../phpdoc.xml  -d application
Collecting files .. OK
Initializing parser .. OK
Parsing files



  [Exception]                                                                      
  No parsable files were found, did you specify any using the -f or -d parameter?  

$mv ../phpdoc.xml ./
$../phpDocumentor-2.8.5/bin/phpdoc parse -d ./application/ -c phpdoc.xml  -d application
Collecting files .. OK
Initializing parser .. OK
Parsing files
Parsing /home/user/directory/example.php ...
```

**Comment by jaapio** [comment]
I don't see how this should improve our project. Most projects have their phpdoc.xml in their root dir. So this would be a major bc break.

**Comment by SJrX** [comment]
I'm foggy about this issue as it has been a year. Arguably the improvement would be that it would be more intuitive for new users as this is non obvious functionality. It's weird that some arguments to the binary take their arguments relative to the current working directory and others to the phpdoc.xml location. 

I mean it works entirely against shell auto completion. It makes sense for arguments in the phpdoc.xml to be relative but command line arguments. That's weird. 

Anyway @jaapio I worked around this issue. I just have a vague recollection  of this being frustrating and non obvious, so I filed a bug.
